roles/telemetry_graphing/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
+=========
+
+A brief description of the role goes here.
```

Comment:
Please update this README to add a description of the role, the requirements, vars, etc